src/aiida/orm/utils/node.py

```diff
@@ -95,7 +95,7 @@ def get_type_string_from_class(class_module, class_name):
     else:
         type_string = f'{class_module}.{class_name}.'
 
-    prefixes = ('aiida.orm.nodes.',)
+    prefixes = ('aiida.orm.nodes.', 'aiida.orm.core.')
```

Comment:
Why?

Reply:
This is related to the changes of adding the `aiida.orm` entry point group and making it possible to load the ORM classes through an entry point. But I will have to go back to the PR and actually test to see if this is still necessary